### ISSUE TITLE
PongoOS USB crash

### ISSUE THREAD

**Issue by justtryingthingsout** [checkra1n/BugTracker#1701]
**Tell us about your setup:**
1. What iDevice are you using? iPhone 6
2. On what version of iOS is it? 12.4.6
3. What version of checkra1n are you using? 0.11.0
4. What is your host system (OS version? Hackintosh? VM? etc.)? Macbook Air
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)? DCSD Cable

**What are the steps to reproduce the issue?**
1. Boot pongoOS using `checkra1n -pvcE`
2. wait for pongoOS to connect to your mac
3. initialize the port first using `termz` tool
4. use issue_cmd.py script from pongoOS scripts

**What do you expect, and what is happening instead?**
I expect it to run a command, but it crashes the usb task in pongoOS, and doesn't even reset it successfully, just stuck until hard reset

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**
yes, it doesn't really matter since I'm not fully booting up

**Any other info, error logs, screenshots, ...?**
![pongoOS USB](https://user-images.githubusercontent.com/31097061/94155311-624b7400-feb1-11ea-8e88-2e3054726f81.png)
![The crash](https://user-images.githubusercontent.com/31097061/94155359-72fbea00-feb1-11ea-8ac4-9ed73f75b535.jpg)

(in case you can't read it from this quality, it says:)
`panic: USB BUG: 0x73656e642034`
`crashed task: usb`
`crashed in required task, resetting..`

**Comment by Siguza** [comment]
Marking this as dupe of https://github.com/checkra1n/pongoOS/issues/6.  
TL;DR: USB doesn't currently work on A7-A8. :/
